### PR TITLE
Extend torch::jit::PropagateInputShapes pass

### PR DESCRIPTION
**Pull request by fortianyou**
Related to #442

### CODE REVIEW

File: pytorch_blade/src/compiler/jit/torch/op_registry.cpp [hunk 1]
```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
```
Comment (Yancey1989) [inline pytorch_blade/src/compiler/jit/torch/op_registry.cpp:1]: do we need any unit test?

Comment (fortianyou) [reply pytorch_blade/src/compiler/jit/torch/op_registry.cpp]: This pull request only adds source files and makes compilation successfully. Unit tests can be added later.

Comment (Yancey1989) [reply pytorch_blade/src/compiler/jit/torch/op_registry.cpp]: better to have an issue to trace that.

**Review by Yancey1989 — APPROVED** [review]
LGTM

**Pull request merged**